tide_search.install

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Install file.
+ */
+
+use Drupal\search_api\Item\Field;
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function tide_search_update_dependencies() {
+  $dependencies['tide_search'][8001] = ['tide_core' => 8007];
+
+  return $dependencies;
+}
+
+/**
+ * Add Path alias field to search index for Topic and Tags.
+ */
+function tide_search_update_8001() {
```

Comment:
Do we need to index the path? Is this because we're going to query to elastic to get the tags for a topic? I think from Slack conversation that's what we're going to do.

Reply:
Given upon a request to `/topic/arts`, FE has 2 options:
* Query BE `/api/v1/route?path=/topic/business` to retrieve the Topic UUID/Name, then query ES to build the tags.
* Directly query ES using `/topic/arts` as `field_topic_path` to build the tags.